mbf_abstract_nav/src/abstract_planner_execution.cpp

```diff
@@ -289,6 +289,7 @@ void AbstractPlannerExecution::run()
 
           plan_mtx_.lock();
           plan_ = plan;
+          // todo compute the cost once!
```

Comment:
You mean, recompute in getCost when cost_ is 0 and the plan doesn't change?
Yes, makes sense

Reply:
yep